src/Libraries/Microsoft.Extensions.AI.Abstractions/Utilities/AIJsonSchemaCreateOptions.cs

```diff
@@ -37,15 +37,24 @@ public sealed record class AIJsonSchemaCreateOptions
     /// </remarks>
     public Func<ParameterInfo, bool>? IncludeParameter { get; init; }
 
+    /// <summary>
+    /// Gets a <see cref="AIJsonSchemaTransformOptions"/> governing transformations on the JSON schema after it has been generated.
+    /// </summary>
+    public AIJsonSchemaTransformOptions? TransformOptions { get; init; }
+
     /// <summary>
     /// Gets a value indicating whether to include the type keyword in inferred schemas for .NET enums.
     /// </summary>
+    [Obsolete("This property has been deprecated.")]
```

Comment:
Note that this property has no equivalent in the the transform options because it is now hardcoded as the default behavior.